usdf/batch.rst

```diff
@@ -103,6 +107,11 @@ condor glide-in shutdown time of 900 seconds or 15 minutes. This means that the
 down after 15 minutes of inactivity (for example, after the workflow is complete), and the glide-in Slurm jobs 
 will exit at that time to avoid wasting idle resources. The ``--dynamic`` option requests that the htcondor slots be dynamic, partionable slots; this is the recommended setting as it supports possible multi-core jobs in the workflow. 
 
+There is support for setting USDF S3DF Slurm account, repo and qos values. By default the account ``rubin`` 
+with a ``default`` repo will be used, and the qos will be ``normal`` by default. If one wants to target a different repo, this is 
+handled as part of the account setting, placed following a colon after the account value proper, 
+e.g., ``--account rubin:commissioning``.
+
```

Comment:
Might need to mention here that the other option for `qos` is `preemptable` and that that is required for the default account `rubin` with no suffix.

Reply:
I added the cautionary note about how preemptable qos can suprise.  That suprise suggests something isn't quite right, but nothing is perfect on the first try.